ocs_ci/deployment/deployment.py

```diff
@@ -216,6 +225,41 @@ def deploy_ocs(self):
                 f"MDS deployment Failed! Please check logs!"
             )
 
+        if config.RUN['cli_params']['monitoring']:
+            # Create a pool, secrets and sc
+            secret_obj = helpers.create_secret(interface_type=constants.CEPHBLOCKPOOL)
+            cbj_obj = helpers.create_ceph_block_pool()
+            sc_obj = helpers.create_storage_class(
+                interface_type=constants.CEPHBLOCKPOOL,
+                interface_name=cbj_obj.name,
+                secret_name=secret_obj.name
+            )
+
+            # Get the list of monitoring pods
+            pods_list = get_all_pods(
+                namespace=defaults.OCS_MONITORING_NAMESPACE,
+                selector=['prometheus', 'alertmanager']
+            )
+
+            # Create configmap cluster-monitoring-config
+            create_configmap_cluster_monitoring_pod(sc_obj.name)
+
+            # Take some time to respin the pod
+            waiting_time = 30
+            logger.info(f"Waiting {waiting_time} seconds...")
+            time.sleep(waiting_time)
```

Comment:
why sleep? can't we wait until pods get into 'running' state?

Reply:
Pod will go to terminate state and then again re-spins. Hence can't use wait_for_resource function.
```
ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: ['oc', '-n', 'openshift-monitoring', '--kubeconfig', '/home/akarsha/aws-install/aug-22/auth/kubeconfig', 'get', 'Pod', 'alertmanager-main-1'].
Error is Error from server (NotFound): pods "alertmanager-main-1" not found
```

Reply:
okay.